tests/Behat/Element/Admin/OrderCreateFormElement.php

```diff
@@ -80,13 +80,39 @@ public function specifyBillingAddress(AddressInterface $address): void
         );
     }
 
-    public function selectShippingMethod(string $shippingMethodName): void
+    public function getAvailableShippingMethods(): array
     {
         $this->clickOnTabAndWait('Shipments & Payments');
 
         $shipmentsCollection = $this->getDocument()->find('css', '#sylius_admin_order_creation_new_order_shipments');
 
+        if (count($shipmentsCollection->findAll('css', '[data-form-collection="item"]')) === 0) {
```

Comment:
if(empty(...)) ?

Reply:
Empty is considered harmful in PHP - see http://php.net/manual/en/function.empty.php#refsect1-function.empty-returnvalues